system/lib/libc/musl/arch/emscripten/bits/alltypes.h

```diff
@@ -86,9 +86,9 @@ typedef long suseconds_t;
 #ifdef __EMSCRIPTEN__
 // For canvas transfer implementation in Emscripten, use an extra 10th control field
```

Comment:
For the source of where this added field comes from, see this commit: https://github.com/kripken/emscripten/commit/1b278917cd325c2aac877e4d09d1a8af07686c9f and in particular, search for the string `_a_transferredcanvases`. That has the value

    // Extra pthread_attr_t field:
    #define _a_transferredcanvases __u.__s[9]

in `system/lib/pthread/library_pthread.c`, which I believe should be updated to `__u.__s[10]` because musl added one field of their own.